### PR TITLE
feat: cart-wishlist done

### PR DESCRIPTION
**Pull request by dev-ashishsethi**

### CODE REVIEW

File: Cart/cart.css [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+    margin-left: 14rem;
+    margin-right: 14rem;
+    
+}
+
+.products{
+    display: flex;
+    flex-direction: column;
+    gap: 2rem;
+    /* margin: 3rem; */
```
Comment (pradipgharge) [inline Cart/cart.css:21]: don't push commented code

File: wishlist/wishlist.css [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    margin-top: 1rem;
+    margin-bottom: 1rem;
+}
+
+.wishlist-container{
+    display: flex;
+    justify-content: center;
+    gap: 2rem;
+    flex-wrap: wrap;
+
```
Comment (pradipgharge) [inline wishlist/wishlist.css:13]: unused space here

**Review by pradipgharge — COMMENTED** [review]
Added a few comments, Rest code looks good. A Lil suggestion you can add a wishlist icon (heart) to the cards.

File: Cart/cart.css [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+    justify-content: center;
+}
+
+.qty-section{
+    display: inline-block;
+    background-color: var(--color-white);
+    margin-left: 1rem;
+}
+
+.qty-number{
```
Comment (SinchanaMS) [inline Cart/cart.css:65]: Keep consistent spacing.
The rest of the code looks good, class names are descriptive and self-explanatory.

**Review by SinchanaMS — APPROVED** [review]

**Review by j24m — APPROVED** [review]
Hello, first things first amazing UI, secondly maybe you are yet to make changes in the nav-bar and footer maybe not, your choice , but maybe you could add a bit more of padding and some margin there, secondly code is readable, most importantly the number of lines of code per file are also fine, a wee bit larger maybe. Good work:)

**Pull request merged**